src/components/Goal.tsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import Subgoals from "./Subgoal";
+import { trpc } from "@/client/lib/trpc";
+import Link from "next/link";
+import styles from "../styles/Home.module.css";
```

Comment:
CSS modules should be scoped to a component, would recommend adding a new module for this component

Reply:
Good point. Added new modules for the components. What is the best way to share styles across components?

Reply:
generally if styles need to be shared across components that's a good indication that there should be a new parent component that encapsulates the shared styles